tests/spec/core/data-type-spec.js

```diff
@@ -133,4 +133,58 @@ describe("Core — data-type attribute", () => {
     );
     expect(displayItems.dataset.type).toBe("sequence");
   });
+
+  it("propagates data-type to subsequent instances of a <var>", async () => {
+    const body = `
+    <section>
+      <p id="a1">First instance of |varA|.</p>
+      <p id="a2">Second instance of |varA: Type1|.</p>
+      <p id="a3">Third instance of |varA|.</p>
+      <p id="a4">Fourth instance of |varA: Type2|.</p>
+      <p id="b1">First instance of |varB:Type1|.</p>
+      <p id="a5">Fifth instance of |varA|.</p>
+    </section>
+    <section>
```

Comment:
Let’s nest this section inside the previous section, so it confirms that nested sections work as expected.

Reply:
Done. :)